js/src/ui/AccountCard/accountCard.js

```diff
@@ -28,6 +28,7 @@ import styles from './accountCard.css';
 export default class AccountCard extends Component {
   static propTypes = {
     account: PropTypes.object.isRequired,
+    allowAddressClick: PropTypes.bool,
```

Comment:
*sigh* Sorry, I setup my auto-complete wrongly. (Happens when you do it late at night.) Updating.